bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -73,6 +74,13 @@
      */
     String PN_DISPLAY_POPUP_TITLE = "displayPopupTitle";
 
+    /**
+     * Name of the resource property that will indicate if the image's link will be opened in a new tab.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.4.1
+     */
+    String PN_LINK_TARGET = "linkTarget";
```

Comment:
@rubnig - I would name these `PN_LINK_TARGET_BLANK` and `linkTargetBlank`.

Reply:
Done